constants.js

```diff
@@ -100,7 +100,7 @@ const checksWhitelist = {
     [checkCompletedEvent]: [ciFailureCheck],
     [issueCommentCreatedEvent]: [respondToReviewCheck]
   },
-  'oppiabot': {
+'oppiabot': {
```

Comment:
Unneeded change.

Reply:
TODO:After failure of frontend tests get fixed.